src/Arcus.Scripting.ApiManagement/Arcus.Scripting.ApiManagement.psm1

```diff
@@ -177,6 +177,54 @@ function Create-AzApiManagementUserAccount {
 
 Export-ModuleMember -Function Create-AzApiManagementUserAccount
 
+<#
+ .Synopsis
+  Create or update users in Azure API Management.
+
+ .Description
+  Create or update users in an existing Azure API Management instance based on a configuration file.
+
+ .Parameter ResourceGroupName
+  The resource group containing the API Management service.
+
+ .Parameter ServiceName
+  The name of the API Management service located in Azure.
+
+ .Parameter ConfigurationFile
+  The file containing the users and their configuration.
+
+ .Parameter StrictlyFollowConfigurationFile
+  Indicates whether the configuration file should strictly be followed, for example remove the user from groups not defined in the configuration file.
+
+ .Parameter ApiVersion
+  [Optional] The version of the api to be used.
```

Comment:
```suggestion
  [Optional] The version of the API to be used.
```

Reply:
Fixed in [`478476f` (#422)](https://github.com/arcus-azure/arcus.scripting/pull/422/commits/478476fd2461b26bcbae63bad3c7c9c7b6fe16b8)